docs/usage/configuration/config_documentation.md

```diff
@@ -23,6 +23,14 @@ followed by a letter. Letters have the following meanings:
 For example, setting `redaction_retention_period: 5m` would remove redacted
 messages from the database after 5 minutes, rather than 5 months.
 
+In addition, configuration options referring to size use the following suffixes:
+
+* `M` = MiB, or 1048576 bytes
+* `K` = KiB, or 1024 bytes 
+
+For example, setting `max_avatar_size: 10M` means that Synapse will not accept files larger than 10485760 bytes
```

Comment:
Could we have commas? (sorry to be difficult)
```suggestion
* `M` = MiB, or 1,048,576 bytes
* `K` = KiB, or 1024 bytes 

For example, setting `max_avatar_size: 10M` means that Synapse will not accept files larger than 10,485,760 bytes
```

Reply:
Sure!